packages/runtime/test/lib/testUtils.ts

```diff
@@ -234,6 +255,11 @@ export async function exchangeMessage(transportServicesCreator: TransportService
     return message;
 }
 
+export async function exchangeMessageWithRequest(sender: TestRuntimeServices, recipient: TestRuntimeServices, request: CreateOutgoingRequestRequest): Promise<MessageDTO> {
```

Comment:
Poor naming. We're not exchanging any messages here. Rather we send a message and wait for its reception.

How about `sendMessageWithRequestAndWaitForReception` or even better, reuse `sendMessageWithRequest` and add a parater `waitForReception`, which will execute the final `syncUntil...`.